### DISCUSSION TITLE
multiValueQueryStringParameters in [@types/aws-lambda]

### DISCUSSION THREAD

**Discussion by ashley-jisc** [DefinitelyTyped/DefinitelyTyped#69967] [Issues with a @types package]
Currently, `multiValueQueryStringParameters` from the `APIGatewayProxyEvent` type from `@types/aws-lambda` is typed like so
```typescript
export interface APIGatewayProxyEventMultiValueHeaders {
    [name: string]: string[] | undefined;
}
```
it should be
```typescript
export interface APIGatewayProxyEventMultiValueHeaders {
    [key: string]: string[] | undefined;
}
```
because you get a type error if you attempt to access anything within the headers that doesn't have a `name` as the value of the key. 

e.g.  `multiValueQueryStringParameters.state` // type error  because it expects `.name` instead of `.state`

[a comment by an automated account is not shown]

**Comment by simonbuchan** [comment]
I don't know what problem you're actually having, but it's not this: the name of the indexer has no effect.
